internal/controllers/cryostat_controller_test.go

```diff
@@ -590,6 +599,12 @@ var _ = Describe("CryostatController", func() {
 						err := t.Client.Get(context.Background(), types.NamespacedName{Name: expectedLink.Name}, link)
 						Expect(kerrors.IsNotFound(err)).To(BeTrue())
 					})
+					It("should remove the application url from APIServer AdditionalCORSAllowedOrigins", func() {
+						apiServer := &configv1.APIServer{}
+						err := t.Client.Get(context.Background(), types.NamespacedName{Name: "cluster"}, apiServer)
+						Expect(kerrors.IsNotFound(err)).To(BeFalse())
+						Expect(len(apiServer.Spec.AdditionalCORSAllowedOrigins)).To(Equal(int(0)))
```

Comment:
You can shorten this with the BeEmpty matcher: https://onsi.github.io/gomega/#beempty

Reply:
After adding an extra entry to the `AdditionalCORSAllowedOrigins` in resources.go, I changed this test case to confirm that the extra entry is still present